sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2IndexesResourceImpl.java

```diff
@@ -133,7 +136,7 @@ public Response dropIndex(
 
     CqlTable table =
         bridge
-            .getTable(keyspaceName, tableName)
+            .getTable(keyspaceName, tableName, false)
             .orElseThrow(() -> new WebApplicationException("Table not found", Status.NOT_FOUND));
```

Comment:
Existing code so no absolute need to change but really should include table name in exception message.
(can be fixed in a later commit -- maybe I should go over REST API and systematically add missing info)